ui/app/services/store.js

```diff
@@ -32,38 +33,28 @@ export function keyForCache(query) {
   return JSON.stringify(cacheKeyObject);
 }
 
-export default Store.extend({
-  // this is a map of map that stores the caches
-  // eslint-disable-next-line
-  lazyCaches: computed({
-    get() {
-      return this._lazyCaches || new Map();
-    },
-    set(key, value) {
-      return (this._lazyCaches = value);
-    },
-  }),
+export default class StoreService extends Store {
+  lazyCaches = tracked(Map);
```

Comment:
I'm curious why this needs to be tracked? Are we binding it to a template somewhere?

Reply:
Good point. Based on what we were doing before (setting the cached value to the new value) I assumed it needed to be tracked, but I'll test without it. I don't think we're accessing this directly in any templates but we are doing some interesting things with the run loop so I wasn't sure if it needed to be ✨ special ✨